# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/certbot/no-download.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- certbot-2.7.4/certbot/setup.py.vanilla	2024-01-07 00:42:17.372588472 +0100
+++ certbot-2.7.4/certbot/setup.py	2024-01-07 00:42:28.564236556 +0100
@@ -24,24 +24,6 @@
 # here to avoid masking the more specific request requirements in acme. See
 # https://github.com/pypa/pip/issues/988 for more info.
 install_requires = [
-    # We specify the minimum acme version as the current Certbot version for
-    # simplicity. See https://github.com/certbot/certbot/issues/8761 for more
-    # info.
-    f'acme>={version}',
-    'ConfigArgParse>=1.5.3',
-    'configobj>=5.0.6',
-    'cryptography>=3.2.1',
-    'distro>=1.0.1',
-    'importlib_resources>=1.3.1; python_version < "3.9"',
-    'importlib_metadata>=4.6; python_version < "3.10"',
-    'josepy>=1.13.0',
-    'parsedatetime>=2.4',
-    'pyrfc3339',
-    'pytz>=2019.3',
-    # This dependency needs to be added using environment markers to avoid its
-    # installation on Linux.
-    'pywin32>=300 ; sys_platform == "win32"',
-    'setuptools>=41.6.0',
 ]
 
 dev_extras = [
